vmm/src/lib.rs

```diff
@@ -762,6 +762,125 @@ impl ReceiveListener {
             }
         }
     }
+
+    fn try_clone(&self) -> std::result::Result<Self, std::io::Error> {
```

Comment:
not about this line but about commit `vmm: handle receiving migration data on additional connections`

`We keep the listening socket around and accept as many questions as
the sender wants to open.` -> probably you mean connections instead of questions?

Reply:
Indeed!